lib/src/lib.rs

```diff
@@ -51,31 +52,29 @@ impl Files {
     }
 
     pub fn get(&self, path: &str) -> io::Result<Cow<'static, [u8]>> {
+        match self.get_raw(path) {
+            Ok((Compression::None, data)) => Ok(data),
+            Ok((Compression::Gzip, compressed)) => {
+                let mut r = GzDecoder::new(Cursor::new(compressed));
```

Comment:
There is no GzDecoder without flate2. I see you've added a #[cfg] in #8, but it really belongs here.